h2o-core/src/main/java/water/util/Log.java

```diff
@@ -9,6 +9,8 @@
 import java.util.ArrayList;
 import java.util.Properties;
 
+import static water.util.StringUtils.fixedLength;
```

Comment:
I thought the goal of this PR is get rid off H2O Log. So why you import the fixedLength method here instead of using this method directly from StringUtils in the case where it is needed?

Reply:
I have moved the method to StringUtils from Log because it was used elsewhere and I wanted to remove the dependency on Log.

Reply:
Oh, I see. So the next step could be to remove Log dependency from all our classes and then this method will not be useful here as well. Cool, thanks.